fiftyone/utils/huggingface.py

```diff
@@ -63,6 +64,26 @@
 
 logger = logging.getLogger(__name__)
 
+def list_hub_datasets():
+    """Lists all FiftyOne datasets available on the Hugging Face Hub.
+
+    This function prints instructions for loading datasets and displays a list of
+    all datasets published by Voxel51 on the Hugging Face Hub.
+    """
+    from huggingface_hub import list_datasets
+    
+    print('You can load any of the following datasets as follows:\n')
```

Comment:
Rather than using `print()` here, please put the example code in the function's docstring, formatted [like this](https://github.com/voxel51/fiftyone/blob/1649f2f2e9a98f7499d08921dbc8e9ac050732c2/fiftyone/core/collections.py#L5528-L5564) so that it is rendered in the docs [like this](https://docs.voxel51.com/api/fiftyone.core.collections.html#fiftyone.core.collections.SampleCollection.limit).